src/components/fields/BooleanField.js

```diff
@@ -49,11 +55,17 @@ if (process.env.NODE_ENV !== "production") {
     onChange: PropTypes.func.isRequired,
     formData: PropTypes.bool,
     required: PropTypes.bool,
+    registry: PropTypes.shape({
+      widgets: PropTypes.objectOf(PropTypes.func).isRequired,
+      fields: PropTypes.objectOf(PropTypes.func).isRequired,
+      definitions: PropTypes.object.isRequired,
+    })
```

Comment:
That'd be possible but we wouldn't benefit from the exclusion at build time (these are defined in an if statement removed when the env is `production`), as we would need to import these def and so so they would be part of the resulting bundle anyway.

It's no big deal having these copied, and it's convenient to have the exact proptypes at hand in each component file anyway.